openmdao/core/tests/test_check_derivs.py

```diff
@@ -2586,6 +2587,10 @@ def test_check_totals_show_progress(self):
         lines = stream.getvalue()
         self.assertFalse("Checking derivatives with respect to" in lines)
 
+        msg = "'_ReprClass' object has no attribute 'write'"
+        with assert_no_warning(AttributeError, msg):
+            prob.check_totals(method='fd', show_progress=True)
```

Comment:
you don't need the assert here.. just doing the check_totals without specifying an `out_stream` suffices to show that it's working..  (it fails with a hard error without your fix... the assert doesn't do anything)

Reply:
Fixed